apps/sms/views/inbox/js/inbox.js

```diff
@@ -639,10 +605,10 @@ var InboxView = {
     }
 
     if (!empty) {
-      TimeHeaders.updateAll('header[data-time-update]');
+      // this.updateContactsInfo();
+      this.list.model = this.listData;
```

Comment:
This changes to `.setModel()` once you update to latest verison of gaia-fast-list.